src/Opdex.Platform.WebApi/Validation/Transactions/TransactionBroadcastNotificationValidator.cs

```diff
@@ -0,0 +1,18 @@
+using FluentValidation;
+using Opdex.Platform.Common.Models;
+using Opdex.Platform.WebApi.Models.Requests.Transactions;
+
+namespace Opdex.Platform.WebApi.Validation.Transactions;
+
+public class TransactionBroadcastNotificationValidator : AbstractValidator<TransactionBroadcastNotificationRequest>
+{
+    public TransactionBroadcastNotificationValidator()
+    {
+        RuleFor(request => request.TransactionHash)
+            .NotEmpty().WithMessage("Transaction hash must not be empty.")
+            .NotEqual(new Sha256()).WithMessage("Transaction hash must be valid.");
```

Comment:
Believe this can be removed. The Sha256 JSON converter will ensure it's not empty and `new Sha256()` is equal to `0000000000000000000000000000000000000000000000000000000000000000` which is theoretically a valid hash. Someone would have to provide that value explicitly because the JSON converter won't convert null/empty to this, it'll instead throw an exception which should get treated as a 400 Bad Request.